packages/api-sdk/src/devices/client/__tests__/metriport.test.ts

```diff
@@ -0,0 +1,28 @@
+import { mocked } from "jest-mock";
+import axios from "axios";
+import { MetriportDevicesApi } from "../metriport";
+
+jest.mock("axios");
+
+describe("getMetriportUserId", () => {
+  const userId = "1";
+  const apiKey = "FAKE_KEY";
```

Comment:
NABD, but its good to make these dynamic with `faker-js` or `uuidv4` or something.